src/paillier.rs

```diff
@@ -35,6 +35,17 @@ impl From<PaillierError> for InternalError {
     }
 }
 
+/// A nonce generated as part of [`PaillierEncryptionKey::encrypt()`].
```

Comment:
todo: Please expand this documentation with the constraints on the nonce. Something like

> A nonce is drawn from the multiplicative group of integers modulo `n`, where `n` is the modulus from the [`PaillierEncryptionKey`].